src/commands/env/log.ts

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import * as Stream from 'stream';
+import * as util from 'util';
+import herokuColor from '@heroku-cli/color';
+import axios from 'axios';
+import { cli } from 'cli-ux';
+import * as Heroku from '@heroku-cli/schema';
+import { Messages } from '@salesforce/core';
+import { Errors, Flags } from '@oclif/core';
+import * as streamUtils from '../../lib/log-stream-utils';
+import { FunctionsFlagBuilder } from '../../lib/flags';
+import Command from '../../lib/base';
+import { resolveAppNameForEnvironment } from '../../lib/utils';
+
+Messages.importMessagesDirectory(__dirname);
+const messages = Messages.loadMessages('@salesforce/plugin-functions', 'env.log.tail');
```

Comment:
```suggestion
const messages = Messages.loadMessages('@salesforce/plugin-functions', 'env.log');
```

And you'll also need to add the new `env.log.md` messages file for this command.